src/generated/resources/schema.graphql

```diff
@@ -115,9 +115,9 @@ type Query {
   "Get all direct connections"
   directConnections: [DirectConnection]!
```

Comment:
I see that other resources, such as Direct Connections, are still required? is it fairly certain we won't run into a similar issue with them?

Reply:
Good question! Yes, I think we should keep the _top-level_ resources, like Connections or CCloud Environments, required. First, we manage connections internally and don't need to consult external APIs for fetching them. Secondly, if we are not able to read connections, it does not make sense to fetch subresources, like Kafka clusters.